src/ape/api/address.py

```diff
@@ -133,6 +136,13 @@ def is_contract(self) -> bool:
 
         return len(HexBytes(self.code)) > 0
 
+    @cached_property
+    def history(self) -> "AccountHistory":
+        """
+        The list of transactions that this account has made on the current chain.
+        """
+        return self.chain_manager.history[self.address]
```

Comment:
I tested the heck out of this. Everything is good, but 1 thing stood out: using a contract.

```
In [18]: [x for x in chain.history[lido_contract]]
Out[18]: []
```

My question is, do we eventually want incoming txns to show here?

Reply:
Transaction history is indexed by nonce, so typically contracts don't have any outgoing transactions